### PR TITLE
Fix issue with registration supplement not rendering

### PR DESCRIPTION
**Pull request by samchrisinger**
# Purpose

Registration supplements were not rendering
# Changes
- Don't index node.registered_meta by schema name
- Fix data-bind in registration viewer to allow undefined question description
- Don't process Node.registered_meta

### CODE REVIEW

**Comment by abought** [comment]
Some tests are failing, related to changes in this PR. Can you please take a look?

File: website/project/model.py [hunk 1]
```diff
@@ -1911,7 +1910,7 @@ def create_draft_registration(self, user, schema, data=None, save=False):
             self.save()
         return draft
 
-    def register_node(self, schema, auth, data, template=None, parent=None):
+    def register_node(self, schema, auth, data, parent=None):
```
Comment (abought) [inline website/project/model.py:1913]: Note to self: remove "template" from the docstring post-merge.

**Comment by abought** [comment]
Note to self: as we change the behavior of `registered_meta`, it's a good idea to document what the expected structure of this dictionary field is. Figuring that out is probably the slowest step in working with that field.

**Comment by abought** [comment]
I think this catches all uses, but will double check when more awake, then merge.

**Comment by abought** [comment]
Possible, but speculative, cases where scripts not mentioned in this PR may be affected by the change to registered_meta. Can you take a look and let me know if these are ok?
- https://github.com/abought/osf.io/blob/9b4053dde1cd1baef2a71adbeb7ba1338a2b5093/scripts/consistency/find_bad_registrations.py#L27-27
- https://github.com/abought/osf.io/blob/a54506d09af3d50f67cc5a5a7d830f335db71a06/website/templates/project/retracted_registration.mako#L31-31

**Comment by samchrisinger** [comment]
@abought I have addressed the second instance (thank you for the catch).

Regarding the first, updating the script is easy, but pointless unless we actually use that code. I have pinged Steve asking this question,

**Comment by abought** [comment]
Comments addressed. Merging. Thanks!

**Pull request merged**